src/components/UserDropDown/UserDropDown.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react';
+
+import { UserList } from '../UserList';
+import { SearchField } from '../SearchField';
+import { Person } from '../../types/Person';
+
+type TUserDropDownProps = {
+  searchField: string;
+  handleInputField: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleUserSelect: (selectedPerson: Person) => void;
+  setHasFocusField: (value: boolean) => void;
+  hasFocusField: boolean;
+  users: Person[];
```

Comment:
Looks like you have too many properties - this is a sign of a bad design

You should provide externally only the properties that are external for the component
For instance, you can entirely handle focus and input inside the component

All it really needs is a collection of people and some callback when the person is selected